packages/ui/src/modal/list/PriceBreakdown.tsx

```diff
@@ -0,0 +1,215 @@
+                amount={profit}
+                address={currency.contract}
+                symbol={currency.symbol}
+                textStyle="h6"
+              />
+              {usdPrice ? (
+                <FormatCurrency
+                  amount={profit * (usdPrice || 0)}
+                  style="subtitle3"
+                  color="subtle"
+                />
+              ) : null}
+            </Flex>
+          </Button>
+        }
+      >
+        <Flex direction="column" css={{ gap: '$2', mt: '$3' }}>
+          <Flex justify="between" align="center">
+            <Text style="subtitle2" color="subtle">
+              Total Listing Value
+            </Text>
+            <FormatCryptoCurrency
+              amount={Number(price) * quantity}
+              address={currency.contract}
+              symbol={currency.symbol}
+              textStyle="subtitle2"
+              textColor="subtle"
+            />
+          </Flex>
+          {collection?.royalties?.bps ? (
+            <Flex justify="between" align="center">
+              <Flex align="center" css={{ gap: '$2' }}>
+                <Text style="subtitle2" color="subtle">
+                  Creator Royalties
+                </Text>
+                <InfoTooltip
+                  side="right"
+                  width={200}
+                  content={
+                    'A fee on every order that goes to the collection creator.'
+                  }
+                />
+              </Flex>
+
+              <Flex align="center" css={{ gap: '$1' }}>
+                <Text style="subtitle2" color="subtle">
+                  -
+                </Text>
+                <FormatCryptoCurrency
+                  amount={
+                    quantity *
+                    Number(price) *
+                    (collection?.royalties?.bps || 0) *
+                    0.0001
+                  }
+                  address={currency.contract}
+                  symbol={currency.symbol}
+                  textStyle="subtitle2"
+                  textColor="subtle"
+                />
+              </Flex>
+            </Flex>
+          ) : null}
+          {marketplace?.fee?.bps ? (
+            <Flex justify="between" align="center">
+              <Text style="subtitle2" color="subtle">
+                Marketplace Fee
+              </Text>
+
+              <Flex align="center" css={{ gap: '$1' }}>
+                <Text style="subtitle2" color="subtle">
+                  -
+                </Text>
+                <FormatCryptoCurrency
+                  amount={
+                    quantity * Number(price) * (marketplace?.fee?.percent / 100)
+                  }
+                  address={currency.contract}
+                  symbol={currency.symbol}
+                  textStyle="subtitle2"
+                  textColor="subtle"
+                />
+              </Flex>
+            </Flex>
+          ) : null}
+          <Span
```

Comment:
if this is a divider maybe worth making it a `hr` and potential making it a primitive to use